models/articles-model.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+
+const fetchArticleById = (articleId) => {
+    if (isNaN(articleId)) {
+      return Promise.reject({ msg: "Bad request" });
+    }
+    return db
+      .query(`SELECT * FROM articles WHERE article_id = $1;`, [articleId])
+      .then((data) => {
+        const articles = data.rows;
+        if (articles.length === 0) {
+          return Promise.reject({ msg: "Not found" });
+        }
+        return articles;
+      });
+  };
+
```

Comment:
- `isNaN` can have some interesting behaviours but I can see what you're doing here and handling the rejection. 
- Consider extending your Promise.reject object to contain `status` property with the relevant status code, this will extend functionality of the error handling middleware capability, see these notes: https://notes.northcoders.com/courses/js-back-end/error-handling#custom-errors
- Good sql statement
- Great to see you use parameterised queries.